common-o2.sh

```diff
@@ -0,0 +1,48 @@
+package: Common-O2
+version: "%(tag_basename)s"
+tag: v0.2.0
+requires:
+  - boost
+  - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - CMake
+source: https://github.com/AliceO2Group/Common
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/sh
```

Comment:
Even if it is ignored, I'd say `#!/bin/bash -ex` which is what is actually being run.